src/App.tsx

```diff
@@ -1,9 +1,16 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setSearch] = useState('');
+  const visibleMovie = moviesFromServer
+    .filter(({ title, description }) => (
+      title.toLowerCase().includes(query.toLowerCase().trimLeft())
+      || description.toLowerCase().includes(query.toLowerCase().trimLeft())
```

Comment:
```suggestion
      title.toLowerCase().includes(query.toLowerCase().trim())
      || description.toLowerCase().includes(query.toLowerCase().trim())
```

Reply:
test failed because not trim spaces after query
![Screenshot from 2023-03-13 12-27-05](https://user-images.githubusercontent.com/33029864/224675794-27a2b13d-ef33-4c14-91f8-0a139f228849.png)